deploy-agent/deployd/common/stats.py

```diff
@@ -205,14 +205,17 @@ def serialize(self):
 
     def _deserialize(self):
         """ read in json, setting defaults for a stat
-            return: None
+            return: bool
```

Comment:
I think there is a better way to architect this by using try/except, but it's more of a python nit.